collections/_article/release-candidate-godot-4-1-4-and-4-2-2-rc-1.md

```diff
@@ -0,0 +1,129 @@
+---
+title: "Release candidate: Godot 4.1.4 RC 1 & 4.2.2 RC 1"
```

Comment:
```suggestion
title: "Release candidates: Godot 4.1.4 RC 1 & 4.2.2 RC 1"
```

Reply:
This was deliberate, but I can change it.